tests/Integration/PackagesListBuilderTest.php

```diff
@@ -24,131 +26,141 @@ public function testBase(): void
         ]);
 
         $this->assertMergePlan([
-            'params' => [
-                '/' => [
-                    'config/params.php',
-                    '?config/params-local.php',
+            Options::DEFAULT_ENVIRONMENT => [
+                'params' => [
+                    Options::DEFAULT_ENVIRONMENT => [
```

Comment:
Seems here `Options::DEFAULT_ENVIRONMENT` not environment. This is application files for config. May be create new constant for this?